docs/contribute.rst

```diff
@@ -351,6 +351,25 @@ good to include respective documentation. You can add or update
 the :ref:`specification`, extend the :ref:`examples` or write a
 new chapter for the user :ref:`guide`.
 
+tmt documentation is written with `reStructuredText`__ and built
+with `Sphinx`__. Various features of both reST and Sphinx are used
+widely in tmt documentation, from inline markup to references. Feel
+free to use them as well to link new or updated documentation to relevant
+parts, to highlight important points, or to provide helpful examples.
+
+A couple of best practices when updating documentation:
```

Comment:
This says "couple of best practices", but is followed by only one bullet point. Also the bullet point starts with lowercase letter and ends with dot, which afaik is not good?

Reply:
* changed to upper-case in 477d5926, no problem.
* I would like to keep "practices" because I hope we could add more points - any quick idea what to ask contributors when it comes to docs?

Reply:
No quick ideas here

Reply:
In 69bc8908 I proposed a couple more points I collected in https://github.com/teemtee/tmt/issues/2657 originally.